src/resolvers-cassandra/Tiles/queries.js

```diff
@@ -9,12 +9,12 @@ const { trackEvent } = require('../../clients/appinsights/AppInsightsClient');
 const { makeSet } = require('../../utils/collections');
 
 /**
- * @param {{tilex: number, tiley: number, tilez: number, avgsentiment: number, mentioncount: number}} rows
+ * @param {{tilex: number, tiley: number, tilez: number, avgsentimentnumerator: number, mentioncount: number}} rows
  */
 function computedtileToTile(row) {
   const coordinates = [geotile.longitudeFromColumn(row.tiley, row.tilez), geotile.latitudeFromRow(row.tilex, row.tilez)];
   const mentionCount = row.mentioncount;
-  const neg_sentiment = row.avgsentiment;
+  const neg_sentiment = row.avgsentimentnumerator / row.mentioncount;
```

Comment:
Division by zero is safe(-ish) in Javascript (just gives you infinity), but sure, let's make it explicit: 62c0ee2